content/en/resources/2024-10-25-product-development-in-ai-era.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Product Development in the AI Era"
+slug: "product-development-in-ai-era"
+draft: false
+event_date: "2024-10-25T12:00:00-05:00"
+image: "img/resources/webinars/2024-10-25-product-development-in-ai-era.png"
+name: "Product Development in the AI Era"
+description: "Video coming soon!"
+events: ['Webinar']
+registration_link: https://rtnl.link/IgJbclLEsfo
+call_to_action: "Register Now"
+video_link: 
+categories: ['Video']
+presenters: ['Edwin Schmierer']
```

Comment:
When this is updated to include the video link, the other presenters will be listed as well. An update will be required to display guest speakers.